Assets/Mirror/Components/NetworkTransformBase.cs

```diff
@@ -39,6 +39,16 @@ public enum Compression { None, Much, Lots, NoRotation }; // easily understandab
         // This component could be on the player object or any object that has been assigned authority to this client.
         bool isClientWithAuthority => hasAuthority && clientAuthority;
 
+        // Sensitivity is added for VR where human players tend to have micro movements so this can quiet down
+        // the network traffic.  Additionally, rigidbody drift should send less traffic, e.g very slow sliding / rolling.
+        [Header("Sensitivity")]
+        [Tooltip("Changes to the transform must exceed these values to be transmitted on the network.")]
+        public float localPositionSensitivity = .01f;
```

Comment:
ok, one more thought: what if we initialize to Mathf.Epsilon?
that's what the old code used too, and people can still modify it if they want to

Reply:
Disagree.  Mathf.Epsilon is too tiny for the default to do any real good.
```
The smallest value that a float can have different from zero.
```